pages/recover.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable indent */
```

Comment:
can we make this as component and add it to the dynamic auth , like this we can just call that page and always have access to reset password